### ISSUE TITLE
Bat flies and Laboulbeniales with metadata

### ISSUE THREAD

**Issue by gbif-portal** [gbif/data-mobilization#210]
## Bat flies and Laboulbeniales with metadata

Dataset link: https://docs.google.com/spreadsheets/d/1-ssEk0zM6s4J2Ug80Gih9iOmXe_D-QQDaD4MtazhqUs/edit?usp=sharing

Region: Neotropics and Europe

Taxon: Fungi: Laboulbeniomycetes, Laboulbeniales and Diptera: Streblidae, Nycteribiidae

Type: metadata

Why is this important: This dataset gives an overview of 12,300 bat flies with their associated metadata, including absence/presence of Laboulbeniales ectoparasitic fungi.

Priority: high

License: CC-BY-NC 4.0

Bibliographic reference: In preparation.

Comments: I am at this stage inquiring if this would be a dataset that could be hosted on GBIF. I am preparing a manuscript for Scientific Data where I and collaborators introduce this dataset as a tool to study interactions and biotic and abiotic traits affecting these hyperparasitic interactions. Do we submit the data to GBIF, or does GBIF provide a link to the dataset that is available elsewhere? I am a little confused about this. Right now, the dataset is presented as a Google Spreadsheet but this is not the format I am envisioning to share this with the world. Any thoughts regarding this would be helpful.

Dataholders contact information: danny.haelewaters@gmail.com

Users contact info: danny.haelewaters@gmail.com

**Comment by dschigel** [comment]
This sounds like a very nice dataset and yes, you are very welcome to publish these data through GBIF before working on the data paper for Scientific Data. We encourage "data first" model on preparing the data papers. You will find basic data publishing instructions [here](https://www.gbif.org/publishing-data). As GBIF was not initially set to support species interactions data, you might like to present a bat individual and all their fungi as co-occurring species within the sampling event, or use the less documented taxa in the system as attributes of the "main" taxa. In the latter case you can flip the same data in two ways: you can make a dataset of fungi and then indicate bats are host, or prepare the dataset of bats, when fungal records are associatedTaxa. You data structure should help you decide what is most adequate. In any case, data publishing requires standartisation to ensure that your field structure are compatible internationally. Most people use Darwin Core archive. This will help you to register your dataset also outside GBIF, e.g. in GloBI (not sure of Sci Data already recommends them in the list of repos & indexes).

**Comment by CecSve** [comment]
@dschigel did you send this reply by email as well and was there then any follow-up?

**Comment by dannyhaelewaters** [comment]
Yes, I did receive an email from @dschigel. I haven't followed up on this myself because I am not yet sure how and where to present the data. We also have our own website, so I have been thinking to keep a record of the dataset along with Google Maps etc. on there, but that should not stop us from publishing these data in GBIF, I suppose?

**Comment by CecSve** [comment]
Sorry for not getting back to you sooner @dannyhaelewaters. Yes, you can have your dataset on your own webpage and still publish to GBIF. If you require any assistance in data publishing on GBIF.org, me and my colleagues at helpdesk@gbif.org can help get you started and provide support. 

You can also familiarize yourself with our [quick-start guide](https://www.gbif.org/publishing-data), read about how [data hosting](https://www.gbif.org/article/4qfLORxmM8kYOIwSYSMc2M/data-hosting) works, and register to [become a publisher](https://www.gbif.org/become-a-publisher).
